nitransforms/io/afni.py

```diff
@@ -119,5 +124,29 @@ def from_string(cls, string):
         return _self
 
 
+class AFNIDisplacementsField(DisplacementsField):
+    """A data structure representing displacements fields."""
+
+    @classmethod
+    def from_image(cls, imgobj):
+        """Import a displacements field from a NIfTI file."""
+        _hdr = imgobj.header.copy()
```

Comment:
leading underscores in these variable names should not be necessary

Reply:
I think that is in @oesteban code?

Reply:
yep - though I'm not sure why. we can chat more about this down the line, so don't worry about it 😄